Running gRPC-Web-Example Failed

OS: Arch LInux

```s
$ node --version
v9.8.0
$ dep version
dep:
 version     : v0.4.1
 build date  : 2018-01-24
 git hash    : 37d9ea0a
 go version  : go1.9.1
 go compiler : gc
 platform    : linux/amd64
$ go version
go version go1.9.2 linux/amd64
```


```s
$ npm install
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

up to date in 5.629s
```

```s
$ npm run get_go_deps

> grpc-web-example@0.1.0 get_go_deps /home/user/go/src/github.com/improbable-eng/grpc-web/example
> ./get_go_deps.sh

Installing go dependencies...
Installing Protobuf to Golang compiler...
```

```s
$ npm start

> grpc-web-example@0.1.0 start /home/user/go/src/github.com/improbable-eng/grpc-web/example
> npm run build:proto && concurrently --kill-others "go run go/exampleserver/exampleserver.go" "npm run webpack-dev"


> grpc-web-example@0.1.0 build:proto /home/user/go/src/github.com/improbable-eng/grpc-web/example
> ./protogen.sh

Optional env var GOBIN is not set; using default derived from GOPATH as: "/home/user/go/bin"
Compiling protobuf definitions
[1] 
[1] > grpc-web-example@0.1.0 webpack-dev /home/user/go/src/github.com/improbable-eng/grpc-web/example
[1] > cd ts && webpack-dev-server --colors --watch --hot --inline --port 8081 --host 0.0.0.0 --output-public-path=http://localhost:8081/build/
[1] 
[1] Project is running at http://0.0.0.0:8081/
[1] webpack output is served from http://localhost:8081/build/
[1] ts-loader: Using typescript@2.2.1 and /home/user/go/src/github.com/improbable-eng/grpc-web/example/ts/tsconfig.json
[0] # command-line-arguments
[0] go/exampleserver/exampleserver.go:42:16: wrappedServer.ServeHTTP undefined (type *grpcweb.WrappedGrpcServer has no field or method ServeHTTP, but does have ServeHttp)
[0] go run go/exampleserver/exampleserver.go exited with code 2
--> Sending SIGTERM to other processes..
[1] npm run webpack-dev exited with code 0
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! grpc-web-example@0.1.0 start: `npm run build:proto && concurrently --kill-others "go run go/exampleserver/exampleserver.go" "npm run webpack-dev"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the grpc-web-example@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2018-04-19T14_23_17_217Z-debug.log
```

Why? Thanks a lot.

--- a/example/go/Gopkg.lock
+++ b/example/go/Gopkg.lock
@@ -2,50 +2,110 @@
 
 
 [[projects]]
-  branch = "master"
   name = "github.com/golang/protobuf"
-  packages = ["proto","ptypes","ptypes/any","ptypes/duration","ptypes/timestamp"]
-  revision = "1e59b77b52bf8e4b449a57e6f79f21226d571845"
+  packages = [
+    "proto",
+    "ptypes",
+    "ptypes/any",
+    "ptypes/duration",
+    "ptypes/timestamp"
+  ]
+  revision = "925541529c1fa6821df4e44ce2723319eb2be768"
+  version = "v1.0.0"
+
+[[projects]]
+  name = "github.com/gorilla/websocket"
+  packages = ["."]
+  revision = "ea4d1f681babbce9545c9c5f3d5194a789c89f5b"
+  version = "v1.2.0"
 
 [[projects]]
   name = "github.com/improbable-eng/grpc-web"
   packages = ["go/grpcweb"]
-  revision = "7174732598544bdecfa12fb81ca2f1601cf1b587"
-  version = "0.3.0-beta"
+  revision = "58d1f6c3d97b6d81d7eebd7d26d5942a1d40cb46"
+  version = "0.6.0"
 
 [[projects]]
   name = "github.com/rs/cors"
   packages = ["."]
-  revision = "7af7a1e09ba336d2ea14b1ce73bf693c6837dbf6"
-  version = "v1.2"
+  revision = "feef513b9575b32f84bafa580aad89b011259019"
+  version = "v1.3.0"
 
 [[projects]]
   branch = "master"
   name = "golang.org/x/net"
-  packages = ["context","http2","http2/hpack","idna","internal/timeseries","lex/httplex","trace"]
-  revision = "d866cfc389cec985d6fda2859936a575a55a3ab6"
+  packages = [
+    "context",
+    "http/httpguts",
+    "http2",
+    "http2/hpack",
+    "idna",
+    "internal/timeseries",
+    "lex/httplex",
+    "trace"
+  ]
+  revision = "5f9ae10d9af5b1c89ae6904293b14b064d4ada23"
 
 [[projects]]
-  branch = "master"
   name = "golang.org/x/text"
-  packages = ["collate","collate/build","internal/colltab","internal/gen","internal/tag","internal/triegen","internal/ucd","language","secure/bidirule","transform","unicode/bidi","unicode/cldr","unicode/norm","unicode/rangetable"]
-  revision = "eb22672bea55af56d225d4e35405f4d2e9f062a0"
+  packages = [
+    "collate",
+    "collate/build",
+    "internal/colltab",
+    "internal/gen",
+    "internal/tag",
+    "internal/triegen",
+    "internal/ucd",
+    "language",
+    "secure/bidirule",
+    "transform",
+    "unicode/bidi",
+    "unicode/cldr",
+    "unicode/norm",
+    "unicode/rangetable"
+  ]
+  revision = "f21a4dfb5e38f5895301dc265a8def02365cc3d0"
+  version = "v0.3.0"
 
 [[projects]]
   branch = "master"
   name = "google.golang.org/genproto"
   packages = ["googleapis/rpc/status"]
-  revision = "a8101f21cf983e773d0c1133ebc5424792003214"
+  revision = "7fd901a49ba6a7f87732eb344f6e3c5b19d1b200"
 
 [[projects]]
   name = "google.golang.org/grpc"
-  packages = [".","balancer","balancer/roundrobin","codes","connectivity","credentials","encoding","grpclb/grpc_lb_v1/messages","grpclog","internal","keepalive","metadata","naming","peer","resolver","resolver/dns","resolver/passthrough","stats","status","tap","transport"]
-  revision = "e687fa4e6424368ece6e4fe727cea2c806a0fcb4"
-  version = "v1.8.2"
+  packages = [
+    ".",
+    "balancer",
+    "balancer/base",
+    "balancer/roundrobin",
+    "codes",
+    "connectivity",
+    "credentials",
+    "encoding",
+    "encoding/proto",
+    "grpclb/grpc_lb_v1/messages",
+    "grpclog",
+    "internal",
+    "keepalive",
+    "metadata",
+    "naming",
+    "peer",
+    "resolver",
+    "resolver/dns",
+    "resolver/passthrough",
+    "stats",
+    "status",
+    "tap",
+    "transport"
+  ]
+  revision = "d11072e7ca9811b1100b80ca0269ac831f06d024"
+  version = "v1.11.3"
 
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "9b63a28eff00bb27eaeb856141b43834ef58214d86c906d4220b8cd794db33c0"
+  inputs-digest = "7c4005512c7bcf970b40141c089e3e607d96b953116641c4635299f5a35acdb7"
   solver-name = "gps-cdcl"
   solver-version = 1

--- a/example/go/Gopkg.toml
+++ b/example/go/Gopkg.toml
@@ -1,4 +1,3 @@
-
 # Gopkg.toml example
 #
 # Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
@@ -17,6 +16,31 @@
 #   source = "github.com/myfork/project2"
 #
 # [[override]]
-#  name = "github.com/x/y"
-#  version = "2.4.0"
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
 
+
+[[constraint]]
+  name = "github.com/golang/protobuf"
+  version = "1.0.0"
+
+[[constraint]]
+  name = "github.com/improbable-eng/grpc-web"
+  version = "0.6.0"
+
+[[constraint]]
+  branch = "master"
+  name = "golang.org/x/net"
+
+[[constraint]]
+  name = "google.golang.org/grpc"
+  version = "1.11.3"
+
+[prune]
+  go-tests = true
+  unused-packages = true

